roles/splunk_cluster_master/tasks/enable_smartstore.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Create indexes.conf conditionally
+  copy:
```

Comment:
I'd go with the `file` module here if all you're doing is making sure a file needs to exist? Otherwise when this re-runs, you're constantly rewriting this indexes.conf

Reply:
Actually I can delete the block because ini_file creates the file if it doesn't exist already.